.github/workflows/release.yml

```diff
@@ -26,6 +26,7 @@ jobs:
         uses: actions/checkout@v2
         with:
           ref: main
+          token: ${{ secrets.GHPAT_FOR_PUSH_RELEASE }}
```

Comment:
Could you create another PR like `ci: xxx` for this change?

Reply:
oh sorry. 
this is unrelated change.